alws/config.py

```diff
@@ -68,6 +68,12 @@ class Settings(BaseSettings):
     sentry_dsn: Optional[str] = None
     sentry_traces_sample_rate: float = 0.2
 
+    github_integration_enabled: bool = False
```

Comment:
Question, should these vars be included into albs-deploy?

Reply:
Of course

Reply:
created https://github.com/AlmaLinux/build-system/issues/215